source/includes/mongodb-compatibility-table-motor.rst

```diff
@@ -5,62 +5,109 @@
    :class: compatibility
 
    * - Motor (Python async) Driver
+     - MongoDB 4.4
+     - MongoDB 4.2
+     - MongoDB 4.0
      - MongoDB 3.6
      - MongoDB 3.4
      - MongoDB 3.2
      - MongoDB 3.0
      - MongoDB 2.6
 
+   * - 2.1
+     -
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     -
+
+   * - 2.0
+     -
+     -
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - 
+
```

Comment:
Please add Motor 1.3 here. It supports the same versions of MongoDB as Motor 1.2.